### PR TITLE
[HTTP/3] Added content types to docs

### PR DESCRIPTION
**Pull request by dcpena**
Added PCX content types to docs

Addresses [PCX-1669](https://jira.cfops.it/browse/PCX-1669)

### CODE REVIEW

File: products/http3/src/content/release.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 order: 6
+pcx-content-type: reference
```
Comment (kodster28) [inline products/http3/src/content/release.md]:
```suggestion
pcx-content-type: interim
```

Comment (kodster28) [reply products/http3/src/content/release.md]: I think we should have a `release-notes` type in the future, but we don't have any definitions yet

**Comment by kodster28** [comment]
Maybe all the chrome + quiche + edge + firefox pages are `how-to`?

**Review by kodster28 — APPROVED** [review]

**Comment by dcpena** [comment]
> Maybe all the chrome + quiche + edge + firefox pages are `how-to`?

Works for me! Like I said, I struggled categorizing these because there wasn't a whole lot of info for one or the other, haha

**Pull request merged**